diff -Nuar glacier-cmd-9999.orig/setup.py glacier-cmd-9999/setup.py
--- glacier-cmd-9999.orig/setup.py	2017-07-02 16:33:00.598219836 +0200
+++ glacier-cmd-9999/setup.py	2017-07-02 16:33:24.021313632 +0200
@@ -34,7 +34,7 @@
           'boto',
           'python-dateutil',
           'pytz',
-          'argparse',
+          'configargparse',
           'prettytable'
       ],
     entry_points="""
